jasp/jasp.py

```diff
@@ -473,14 +474,15 @@ def exception_handler(context_manager, etype, evalue, traceback):
     False if not.
 
     """
+    cwd = context_manager.vaspdir
```

Comment:
They should be the same. I had trouble keeping one of the lines below 80 characters without assigning this to a variable. Namely, the new line 516 below.